include/api.php

```diff
@@ -3365,17 +3363,6 @@ function api_ff_ids($type,$qtype)
 
 	$user_info = api_get_user($a);
 
-	if ($qtype == 'friends') {
```

Comment:
Why did you remove this code?

Reply:
Ha, saw the commit log, never mind.

Reply:
The `$sql_extra` variable is not used anywhere in the function.